### PR TITLE
[#66] update zgw consumers

### PR DESCRIPTION
**Pull request by SonnyBA**
Fixes the open-notificaties part of https://github.com/maykinmedia/open-api-framework/issues/66

### CODE REVIEW

**Review by stevenbal — CHANGES_REQUESTED** [review]
I'm not sure why this happens, but despite not having a Service linked to AutorisatiesConfig, it does say that there is a service configured for it:

image

image

There is a Service for autorisaties, but it doesn't have the correct type:
image

I'm not sure if this was misconfigured on my end or if this is due to data migrations?

**Comment by SonnyBA** [comment]
> I'm not sure why this happens, but despite not having a Service linked to AutorisatiesConfig, it does say that there is a service configured for it:
> 
> image
> 
> image
> 
> There is a Service for autorisaties, but it doesn't have the correct type: image
> 
> I'm not sure if this was misconfigured on my end or if this is due to data migrations?

Are you sure you've had the latest `commonground-api-common` installed in your virtual env? Whenever a branch is pinned you might need to force it to reinstall to the latest commit. There were some changes done to the config view in `44a40320f9a6f3506bbd1f7a9d2a2c117a528fb5`. Testing the view with a `AuthorizationsConfig` without a service linked shows a cross (instead of a checkmark):
Screenshot_20241115_112137
Screenshot_20241115_112013

**Comment by SonnyBA** [comment]
> I'm not sure why this happens, but despite not having a Service linked to AutorisatiesConfig, it does say that there is a service configured for it:
> 
> image
> 
> image
> 
> There is a Service for autorisaties, but it doesn't have the correct type: image
> 
> I'm not sure if this was misconfigured on my end or if this is due to data migrations?

Regarding the last comment, I haven't been able to reproduce this behavior. I tested the function that tries to determine the `api_type` for the `Service` with the URL shown in the screenshot:
```
>>> _get_api_type("http://localhost:8000/autorisaties/api/v1/")
APITypes.ac
>>> _get_api_type("localhost:8000/autorisaties/api/v1/")
APITypes.ac
```

**Comment by stevenbal** [comment]
> Are you sure you've had the latest commonground-api-common installed in your virtual env

@SonnyBA you might be right, I tested it again and it works now :+1:

File: CHANGELOG.rst [hunk 1]
```diff
@@ -2,6 +2,20 @@
 Changes
 =======
 
+1.x.x (TBD)
+-----------
+**Bugfixes and QOL**
+
+* [#66] updated commonground-api-common to 0.x.x (TBD)
```
Comment (stevenbal) [inline CHANGELOG.rst:9]: Should probably add a line to mention zgw-consumers being upgraded to 0.35.1 as well
```suggestion
* [#66] updated commonground-api-common to 2.1.0 (TBD)
```

**Review by stevenbal — APPROVED** [review]

File: requirements/base.in [hunk 2]
```diff
@@ -5,5 +5,4 @@ markdown
 self-certifi
 furl
 
-# API libraries
-gemma-zds-client
+commonground-api-common>=2.0.1
```
Comment (annashamray) [inline requirements/base.in:8]: This line feels unnecessary since we still pin `commongground-api-common` dependencies in "requirements/*.txt" files

**Review by annashamray — APPROVED** [review]
:+1:

File: requirements/base.in (hunk 2 above)
Comment (stevenbal) [reply requirements/base.in:8]: using `bin/compile_dependencies.sh -P commonground-api-common` should work as well

**Pull request merged**